### ISSUE TITLE
changelog

### ISSUE THREAD

**Issue by dwelle** [excalidraw/excalidraw#2584]
We should either remove `/CHANGELOG.md` (or just replace the contents with a link to the other one) in favor of `/src/packages/excalidraw/CHANGELOG.md`, or the other way round.

**Comment by ad1992** [comment]
@dwelle @lipis as the first step let's integrate the Github action for semantic pr title?

**Comment by dwelle** [comment]
> as the first step let's integrate the Github action for semantic pr title?

yes, let's start with that. Btw, if possible, I'd choose as few of the commit types as possible. For example, I'm not sure if distinguishing `refactor` and `chore` adds any benefit. Ideally I'd also merge `improvements` into some other type, but not sure.

**Comment by lipis** [comment]
I would stick to this list and not reinventing new terms: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json

`chore` for example is all the dependency updates, translations and others, that is not actually a `refactor`

**Comment by ad1992** [comment]
Yep, +1, better to follow an existing convention (also some users will be familiar with it)  and these would be supported in the Github action out of the box afaik as well so we need not maintain separately.

**Comment by NMinhNguyen** [comment]
Having used conventional commits for changelog generation extensively at work (via Lerna) and recently having tried https://github.com/atlassian/changesets, I'd say that the Changesets flow is preferable because commit messages can't easily be edited once they land on the main branch (you have to rewrite history causing conflicts for others), and generally speaking commit messages (or PR titles) aren't very suitable for authoring text in my opinion. With Changesets, it's a `.md` file that can be edited subsequently (or even deleted) and you can have them auto formatted by Prettier, you can preview the rendered Markdown in VS Code etc. :) Pretty sure you should be able to programmatically customise changelog generation to suit your needs as well.
